docs/man/ntp.toml.5.md

```diff
@@ -198,6 +206,12 @@ time.
     `DENY` kiss code. When set to `false` (the default), normal NTP messages are
     also allowed.
 
+`accept-ntp-versions` = [ `3` | `4` | `5`, .. ] (**[3, 4]**)
+:   An array of NTP versions that are accepted by the server. By default only
+    NTPv3 and NTPv4 packets are supported. To also allow draft NTPv5 packets,
+    you can set this value to `[3, 4, 5]`.
```

Comment:
Perhaps we should add that NTPv5 support is still considered in beta, and that things might not be backwards compatible with future versions.

Reply:
Fixed in a3c075476f93ee85b1735790d2ac90df37f2d190